src/style.css

```diff
@@ -1,3 +1,69 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  box-sizing: border-box;
```

Comment:
`box-sizing` is not inherited from the parent element
so it's better to specify for all elements